src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -76,8 +76,10 @@ namespace ts.codefix {
 
                     case ts.SyntaxKind.Parameter:
                         const functionDeclaration = <FunctionDeclaration>parent.parent;
-                        return [functionDeclaration.parameters.length === 1 ? deleteNode(parent) : deleteNodeInList(parent),
-                            prefixIdentifierWithUnderscore(identifier)];
+                        const deleteAction = functionDeclaration.parameters.length === 1 ? deleteNode(parent) : deleteNodeInList(parent);
+                        return errorCode === Diagnostics.Property_0_is_declared_but_its_value_is_never_read.code

```

Comment:
`isParameterPropertyDeclaration(identifier)` instead of looking at the code.

Reply:
My feeling was that the code might be applied to other symbols in the future but that it's inherently about properties so it was a valid way to perform the check.